PkSolver/CMakeLists.txt

```diff
@@ -4,9 +4,20 @@ set(LIBRARY_NAME PkSolver)
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 
+if(ITK_INSTALL_PREFIX)
```

Comment:
any idea why this needs to be done explicitly? It was not needed in the past

Reply:
perhaps it has to do with the fact that it is a patched down ITK fork

Reply:
yes the include directory structure of the pre-compiled and installed ITK is different from the one in Slicer. Still most includes are found just these ones in vxl/.. not